src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -55,6 +55,20 @@ public ProcessStartInfo(string fileName, string arguments)
             _arguments = arguments;
         }
 
+        /// <devdoc>
+        ///     Specifies the name of the application that is to be started, as well as a set
+        ///     of command line arguments to pass to the application.
+        /// </devdoc>
+        public ProcessStartInfo(string fileName, IEnumerable<string> arguments)
+        {
+            _fileName = fileName;
+
+            foreach (string argument in arguments)
+            {
+                ArgumentList.Add(argument);
+            }
```

Comment:
Instead of adding each argument individually, this could be:
```suggestion
            if (arguments is not null)
            {
                _argumentList = new Collection<string>(new List<string>(arguments));
            }
```
and let `List<T>`'s ctor handle it however makes sense.